packages/SwingSet/test/snapshots/test-xsnap-store.js.md

```diff
@@ -10,25 +10,25 @@ Generated by [AVA](https://avajs.dev).
 
     {
       compressSeconds: 0,
-      hash: 'c77e111a40bba743b29cc5bb3abfbd57c1a7143772117cb01da874f5cfe59415',
+      hash: '290c5278c442f0af9ebdc62674f418da51e99e74382c24c1111be01e16490a53',
       rawSaveSeconds: 0,
-      uncompressedSize: 168123,
+      uncompressedSize: 168379,
     }
 
 > after SES boot - sensitive to SES-shim, XS, and supervisor
 
     {
       compressSeconds: 0,
-      hash: 'fd8d35d908d4d5892fb049edec5867ebd737e21be1bf8a4ae2bd810f81260061',
+      hash: 'caf69d62160347624ec1680d819db5152fd9e766e3f8e3f87c5a1e95ee2c1cbf',
       rawSaveSeconds: 0,
-      uncompressedSize: 801387,
+      uncompressedSize: 724579,
```

Comment:
Impressive how we reduced the heap snapshot size!

Reply:
Yeah, removing the hardened WeakSet is probably what caused this shrinkage.